<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  
  <property name="styleSheet">
   <string>
   QMainWindow {
       background-color: #1e1e1e; /* Dark gray background */
   }
   QMainWindow::title {
    background-color: #333; /* Darker gray background for the title bar */
    color: white; /* Text color for the title bar */
    padding: 10px; /* Padding around the title text */
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
  <property name="styleSheet">
    <string>
    QWidget#centralwidget {
        background-color: #1e1e1e; /* Dark gray background */
    }
    QLabel#input_label, QLabel#output_label, QLabel#prefix_label, QLabel#postfix_label, QPushButton, QLineEdit {
        background-color: #2a2a2a; /* Darker gray background for widgets */
        color: white; /* Text color */
        border: 1px solid #404040; /* Border color */
        border-radius: 4px; /* Rounded corners */
        padding: 4px; /* Padding around the content */
    }
    QLabel#input_label, QLabel#output_label, QLabel#prefix_label, QLabel#postfix_label{
        font-size: 12pt; /* Increase font size */
    }
    QLCDNumber, QPushButton, QLabel#prefix_lcd, QLabel#postfix_lcd, QLabel#error_display {
        color: white; /* Text color */
    }
    QPushButton:hover {
        background-color: #3a3a3a; /* Darker background on hover */
    }
    QPushButton:pressed {
        background-color: #1a1a1a; /* Darker background when pressed */
    }
    QLineEdit {
        border: 1px solid #404040; /* Border color */
    }
    </string>
   </property>
   <widget class="QLineEdit" name="input_text">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>25</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_delete">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>25</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_0">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plus">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_star">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="input_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Input:</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Output:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="output_lcd">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>205</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>9</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="prefix_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>265</y>
      <width>135</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Prefix Notation:</string>
    </property>
   </widget>
   <widget class="QLabel" name="prefix_lcd">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>265</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QLabel" name="postfix_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>325</y>
      <width>135</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Postfix Notation:</string>
    </property>
   </widget>
   <widget class="QLabel" name="postfix_lcd">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>325</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QLabel" name="error_display">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="font">
        <font>
            <pointsize>20</pointsize>
        </font>
    </property>
    <property name="styleSheet">
        <string>* { color: red; }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_equal">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>408</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
